### DISCUSSION TITLE
Biologics (ADC) whole-body PBPK/QSP In Person Workshop @ACoP (10 November 2024)

### DISCUSSION THREAD

**Discussion by nicospinu** [Open-Systems-Pharmacology/Forum#1797] [Announcement]
Dear OSP Community,

We are happy to invite you to our workshop to take place officially as part of the program at the ACoP Conference in Arizona.

The workshop will cover general introductions to applying the OSP Suite, PK-Sim® and MoBi®, and will provide state-of-the-art example models and development workflows. In-depth exercises will cover **PBPK and PD modeling for monoclonal antibodies**, **antibody-drug-conjugates (ADCs)** for tumor growth inhibition, **FcRn inhibitors**, and **bispecific immune cell engages**.

Find more details [here](https://esqlabs.com/event-20240704-2/). 

Please don't hesitate to reach out to us for any further queries! [training@esqlabs.com](mailto:training@esqlabs.com)

We’re looking forward to meeting you at the ACoP 2024 Conference!

Best wishes,
Nicoleta Spinu
Scientist & Platform Lead Training Business @ESQlabs

**Comment by StephanSchaller** [comment]
Register now to secure your seat!
